tools/readme-catalog/gen-audit-catalog.js

```diff
@@ -0,0 +1,62 @@
+import * as fs from 'fs';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+import { getGeneratedStringWithReplacement } from './common.js';
+
+/**
+ * Generates a Cloud Audit Log catalog markdown for the README.
+ * @example Example input:
+ * [{
+ *   "serviceName": "workflows.googleapis.com",
+ *   "displayName": "Workflows",
+ *   "methods": [
+ *     {
+ *       "methodName": "google.cloud.workflows.v1.Workflows.CreateWorkflow",
+ *       "lastAdded": "1607367890"
+ *     },
+ *   ]
+ * },
+ *   ...
+ * ]
+ * {@link ../../json/audit/service_catalog.json}
```

Comment:
Discussion: This is fantastically clear and detailed commenting. Is this complying with any particular commenting standard? I've been meaning to look up best practice around node.js.

Reply:
Thank you.

I am generally using the JSDoc standard, which has support with editors like VS Code:

https://jsdoc.app/
https://code.visualstudio.com/docs/languages/javascript#_jsdoc-support